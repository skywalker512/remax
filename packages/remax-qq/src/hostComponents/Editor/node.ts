export const alias = {
  readOnly: 'read-only',
  placeholder: 'placeholder',
  showImgSize: 'show-img-size',
  showImgToolbar: 'show-img-toolbar',
  showImgResize: 'show-img-resize',
  onReady: 'bindready',
  onFocus: 'bindfocus',
  onBlur: 'bindblur',
  onInput: 'bindinput',
  onStatusChange: 'bindstatuschange',
  animation: 'animation',
};

export const props = Object.values(alias);
