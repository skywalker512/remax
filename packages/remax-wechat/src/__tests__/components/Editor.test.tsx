import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import Editor from '../../hostComponents/Editor';

describe('Editor', () => {
  it('render correctly', () => {
    const testRenderer = TestRenderer.create(<Editor />);

    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
