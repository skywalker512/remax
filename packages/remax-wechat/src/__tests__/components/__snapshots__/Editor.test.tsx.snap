// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Editor render correctly 1`] = `<editor />`;
